bundle/libraries/local_path.go

```diff
@@ -77,6 +80,16 @@ func IsLibraryLocal(dep string) bool {
 	return IsLocalPath(dep)
 }
 
+func IsLocalRequirementsFile(dep string) (string, bool) {
+	dep, ok := strings.CutPrefix(dep, "-r")
+	if !ok {
+		return "", false
+	}
+
+	dep = strings.TrimSpace(dep)
+	return dep, IsLocalPath(dep)
+}
+
```

Comment:
Could you include a small unit test for this function?

Reply:
Added here https://github.com/databricks/cli/pull/2736/commits/ee64e50560029a499a95fb4a4646c3a6811bdcae